modules/rules_pitest/0.0.3/MODULE.bazel

```diff
@@ -0,0 +1,37 @@
+"Bazel dependencies"
+
+module(
+    name = "rules_pitest",
+    version = "0.0.3",
```

Comment:
please use the version `0.0.2.bcr.1`, since `0.0.3` is not an actual release upstream.